src/native/libs/System.Native/pal_uid.c

```diff
@@ -237,3 +237,55 @@ int32_t SystemNative_GetGroups(int32_t ngroups, uint32_t* groups)
 
     return getgroups(ngroups, groups);
 }
+
+int32_t SystemNative_GetUName(uint32_t uid, char* buffer, size_t bufferSize)
+{
+    assert(buffer != NULL);
+    assert(bufferSize > 0);
+
+    if (buffer == NULL || bufferSize <= 0)
+    {
+        errno = EINVAL;
+        return -1;
+    }
+
+    struct passwd u;
+    struct passwd* pu; 
+
+    errno = 0;
```

Comment:
We don't need to set `errno` to `0`, as the error number is returned from `getpwuid_r`.

Reply:
No problem, I can remove it. I did it because I saw a few other cases in `src/native/libs`.